pom.xml

```diff
@@ -5,12 +5,156 @@
+            <scope>system</scope>
+            <version>1.0</version>
+            <systemPath>${basedir}\src\lib\calculator-1.0.jar</systemPath>
+        </dependency>
+    </dependencies>
+
     <properties>
-        <maven.compiler.source>8</maven.compiler.source>
-        <maven.compiler.target>8</maven.compiler.target>
+        <java.version>1.8</java.version>
     </properties>
 
+    <build>
+    <plugins>
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-checkstyle-plugin</artifactId>
+            <version>3.1.2</version>
+            <dependencies>
+                <dependency>
+                    <groupId>com.puppycrawl.tools</groupId>
+                    <artifactId>checkstyle</artifactId>
+                    <version>8.41.1</version>
+                </dependency>
+            </dependencies>
+            <configuration>
+                <configLocation>
+                    <!--  pay attention to use exact url for checkstyle plugin rule set  -->
+                    https://raw.githubusercontent.com/DmitryKhodakovsky/epam-test-auto-spring-2021/master/codestyle_checks.xml
+                </configLocation>
+                <includeTestSourceDirectory>true</includeTestSourceDirectory>
+                <violationSeverity>warning</violationSeverity>
+                <consoleOutput>true</consoleOutput>
+            </configuration>
+            <executions>
+                <execution>
+                    <id>validate</id>
+                    <phase>validate</phase>
+                    <goals>
+                        <goal>check</goal>
+                    </goals>
+                </execution>
+            </executions>
+        </plugin>
+        <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.1</version>
+                <configuration>
+                    <source>${java.version}</source>
+                    <target>${java.version}</target>
+                    <encoding>UTF-8</encoding>
+                </configuration>
+        </plugin>
+
+        <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-surefire-plugin</artifactId>
+            <version>3.0.0-M5</version>
+            <configuration>
+                <suiteXmlFiles>
+                    <suiteXmlFile>${basedir}\src/test/resources/firstTest.xml</suiteXmlFile>
+                    <suiteXmlFile>${basedir}\src/test/resources/secondTest.xml</suiteXmlFile>
+                    <suiteXmlFile>${basedir}\src/test/resources/allTests.xml</suiteXmlFile>
+                </suiteXmlFiles>
+            </configuration>
+        </plugin>
+
+    </plugins>
+</build>
+
+<profiles>
+    <profile>
+        <id>runSumAndSubtractTests</id>
+        <activation>
+            <activeByDefault>true</activeByDefault>
+        </activation>
+        <build>
```

Comment:
Why do you repeat `build` section in the each **maven-profile**? Try to use properties instead of duplication build section